Automattic/jetpack
# Sync: remove product_cat from blocked taxonomies

Fixes https://github.com/Automattic/jetpack/issues/19524.

#### Changes proposed in this Pull Request:
Remove `product_cat` from the list of taxonomies that are not synced by Jetpack Sync.

#### Jetpack product discussion
See @gibrown's comment at pcNPJE-5Y-p2#comment-203.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Do a Full Sync of functions. 
2. Verify the taxonomy blacklist no longer contains product_cat. 
3. Create a new post linked to the taxonomy and verify the post in the cache db has it as a linked taxonomy relationship.
